Added extra check since some eml files where still passing (#14600)
demisto/content: Packs/CommonPlaybooks/ReleaseNotes/2_0_4.md

#### Playbooks
##### Extract Indicators From File - Generic v2

> Added a condition to the task **Is there a text-based file** to check that if a string message/rfc822 does not exist in file.info.

Reviewer: ```suggestion
Added a condition to the **Is there a text-based file** task to verify if a string message/rfc822 does not exist within `file.info`.
```
